sdk/appconfiguration/app-configuration/src/internal/cryptoHelpers.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createHash, createHmac } from "crypto";
+
+/**
+ * @internal
+ * @ignore
+ */
+export function sha256Digest(body: any): Promise<string> {
```

Comment:
in the caller you default body to "", maybe we can tighten the type of body away from any? `unknown` would be better if nothing else.

Reply:
Done - just used `string | undefined` which covers everything.